.istanbul.yml

```diff
@@ -3,4 +3,4 @@ check:
     statements: 90
     lines: 90
     branches: 90
-    functions: 90
+    functions: 80
```

Comment:
The app.js downloadAndProcess... function could be refactored so the parsing could be carried out on a small local file, and the method that tested without the overhead of downloading a large file and processing thousands of rows of data.